playwright/test/work.test.ts

```diff
@@ -33,8 +39,13 @@ test.describe(`Scenario 1: a user wants to see relevant information about where
     await workWithPhysicalLocationOnly(context, page);
     const { whereToFindIt, encoreLink, unavailableBanner } =
       await getWhereToFindItAndEncoreLink(page);
-    expect(whereToFindIt).toBeTruthy();
-    expect(encoreLink || unavailableBanner).toBeTruthy();
+
+    await expect(whereToFindIt).toBeVisible();
+
+    const encoreLinkIsVisible = await encoreLink.isVisible();
+    const unavailableBannerIsVisible = await unavailableBanner.isVisible();
```

Comment:
I think `expect().toBeVisible()` [is now preferred by Playwright](https://playwright.dev/docs/api/class-locator#locator-is-visible) as they say it "avoids flakiness"

Reply:
And then I think it removes the need for `expect().toBe(true)` below?

Reply:
I believe this test is checking that either the login link is visible _or_ there's an unavailable banner, and which it is depends on the state of the real item which could change and inadvertently break this test? 

I can't see a nice way of doing this with other than resolving both promises and `or`ing in the `expect`?

Reply:
I might be wrong but would this cover it? 
https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible

> To check that at least one element from the list is visible, use [locator.first()](https://playwright.dev/docs/api/class-locator#locator-first).

Reply:
✨ Yes, i think so.

Reply:
Note that this requires a playwright upgrade ([`or` is available from v1.33](https://playwright.dev/docs/api/class-locator#locator-or)).

Reply:
Hmmm interesting. It doesn't look like there are problematic breaking changes between `1.30` and `1.33`, but it _is_ extra to this ticket. What do you think, would it be worth trialling?

Reply:
Yes, I think so. I've bumped to 1.4 here (latest) and checked whether I get different flakiness from main and I do not. I will pull the upgrade commits into a separate PR though.

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/pull/10520